mesa/space.py

```diff
@@ -252,7 +259,7 @@ def get_neighbors(
         moore: bool,
         include_center: bool = False,
         radius: int = 1,
-    ) -> List[Coordinate]:
+    ) -> List[GridContent]:
```

Comment:
I have also noticed this mis-annotation. Again, "neighborhood"/"neighbor" to denote either a neighbo coordinate or a content of a neighbor coordinate is confusing. This fix can be put in a separate PR so that it can be merged soon.